tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -52,3 +68,48 @@ def dismiss_tooltip(self):
         confirm_element = Wait(self.marionette).until(expected.element_present(*self._button_popup_intro))
         Wait(self.marionette).until(expected.element_displayed(confirm_element))
         confirm_element.tap()
+
+
+class BugzillaLiteStage(BugzillaLite):
+
+
+    name =  'Bugzilla Lite Stage'
+    manifest_url = 'http://bzlite-staging.herokuapp.com/manifest.webapp'
+
+    def dismiss_content(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._content_locator))))
+        self.marionette.find_element(*self._close_locator).tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._login_form_locator))
+        import time
+        time.sleep(1)
```

Comment:
Why is this time.sleep() call needed? If it's really needed can you comment in here why it's needed? Otherwise try to remove it.